### PR TITLE
added alt prop and imgprops to s3 img component

### PR DESCRIPTION
**Pull request by jacoblogan**
#### Description of changes
Updated S3Img to accept a string alt prop as well as imgProps object to pass through properties to the underlying img tag

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#steps-towards-contributions)
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by wlee221 — CHANGES_REQUESTED** [review]
LGTM! I especially love the addition of `imgProps`, which alone will open up so many use cases including #6396. Just got two nits.

1. Have you run `yarn && yarn build` after making this changes? Two artifacts should be changed from this PR: `components.d.ts` and `amplify-s3-image/README.md`, but I only see the former.
2. see comment above

File: packages/amplify-ui-components/src/components/amplify-s3-image/amplify-s3-image.tsx [hunk 1]
```diff
@@ -32,6 +34,8 @@ export class AmplifyS3Image {
 	@Prop() handleOnLoad: (event: Event) => void;
 	/** Function executed when error occurs for the image */
 	@Prop() handleOnError: (event: Event) => void;
+	/** Attributes to be placed on the img element: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attributes */
+	@Prop() imgProps?: object;
```
[the remark this answers is not in the document]
Comment (jacoblogan) [reply packages/amplify-ui-components/src/components/amplify-s3-image/amplify-s3-image.tsx:38]: yes, good catch. I'll make those changes

**Review by wlee221 — APPROVED** [review]
LGTM! Thanks @jacoblogan 👍

**Comment by iartemiev** [comment]
@jacoblogan please use the Squash and Merge option when merging PRs and squash the title to a [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary)

**Pull request merged**